src/features/codeLensProvider.ts

```diff
@@ -139,4 +141,41 @@ export default class OmniSharpCodeLensProvider extends AbstractProvider implemen
                 { title: "debug test", command: 'dotnet.test.debug', arguments: [testFeature.Data, fileName, testFrameworkName] }));
         }
     }
+
+    private _updateCodeLensForTestClass(bucket: vscode.CodeLens[], fileName: string, node: protocol.Node) {
+        //if the class doesnot contain any method then return
+        if (!node.ChildNodes.find(value => (value.Kind == "MethodDeclaration"))) {
+            return;
+        }
+
+        let testMethodsInClass = new Array<string>();
```

Comment:
Is this the most idiomatic way to declare an empty array in TS? (@TheRealPiotrP )